language_service/src/code_lens/tests.rs

```diff
@@ -92,7 +92,6 @@ fn two_entrypoints() {
             ◉operation Main() : Unit{
             }◉
 
-            @EntryPoint()
```

Comment:
Well, the test is `two_entrypoints`, so if you remove this line, the point of the test is defeated.

Are you sure the existing test doesn't pass anymore? Since you've made the product changes to allow entrypoint errors, I feel like this test should be ok without changes.